src/main/protobuf/v1/relationship.proto

```diff
@@ -30,7 +30,6 @@ message PartyRelationshipV1 {
     SUSPENDED = 3;
     DELETED = 4;
     REJECTED = 5;
-    TOBEVALIDATED = 6;
```

Comment:
ℹ️ remember that you can use position `6` in the future only if there is not data containing the `TOBEVALIDATED` value

Reply:
Yes, is just a rollback to limit the pr scope